.github/workflows/tag.yml

```diff
@@ -0,0 +1,44 @@
+name: Tag
+
+on:
+  push:
+    tags: '*'
+
+jobs:
+  test:
+    name: Tag
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18.7
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Style checks
+        run: |
+          make style
+          if ! git diff --exit-code HEAD; then
+            echo
+            echo "*** Files are not formatted properly. See the above diff for more info."
+            exit 1
+          fi
+      - name: Unit tests
+        run: |
+          make test
+      - name: Build
+        run: |
+          make build
+      - name: Dry run
+        run: |
+          make dry-run
+      - name: Build and Push Container Image
+        run: |
+          docker login -u "${{ secrets.QUAY_RHACS_ENG_RW_USERNAME }}" --password-stdin <<<"${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}" quay.io
```

Comment:
Please make sure somebody besides you have permissions to configure secrets for this repo https://github.com/stackrox/k8s-istio-cve-pusher